resources/i18n/en-US/builtin.ftl

```diff
@@ -1,3 +1,24 @@
 -app-name = Druid
 
 hello-counter = Current value is { $count }
+
+# The 'application' menu on macOS
+
+macos-menu-about-app = About { -app-name }
+macos-menu-preferences = Preferences...
+macos-menu-hide-app = Hide { -app-name }
+macos-menu-hide-others = Hide Others
+macos-menu-show-all = Show All
+macos-menu-services = Services
+macos-menu-application-menu = { -app-name }
+macos-menu-quit-app = Quit { -app-name }
+
+# macOS 'File' menu
+macos-menu-file-menu = File
```

Comment:
So to make Alt-navigation work on Windows, this should be `&File`, `&New File`, etc. But in the current code, that causes the literal ampersand to be shown on mac. There are of course a bunch of ways to handle this, but I personally would lean towards having the ampersands in the strings and stripping them on mac. (I'm fairly sure some previous iteration of the code did this).

Reply:
My plan here is to have different localized strings for windows; this will let us exactly match native localization per-platform.

That said, my intent was that macOS should be stripping `&`, so if it isn't that's a bug.

Reply:
Interesting; I thought about that a bit. My personal feeling is that things should be fanned out cross-platform by default, but there should be the option to override that to make things more platform-native. But if the conventions are consistently different, maybe it's better to explicitly highlight the requirement to do multiple versions.

Reply:
My feeling here is that getting window/app menus right is an important part of feeling native, and the only way to really do that is to declare them per-platform.

For instance: the menu item to quit the application is 'Quit' on mac and 'Exit' on windows. On mac, it's in the 'application' menu; on windows it's in the 'File' menu.

I think it will be hard to have a single 'default menu' that correctly handles this case, and many others like it. Instead what I would like to do is to expose convenience functions that will create individual menus and menu items, per platform, so that it is as easy as possible for the user to create native-feeling menus, but still have the expectation be that you're doing this manually.